lib/ripper-tags/default_formatter.rb

```diff
@@ -31,7 +31,7 @@ def relative_path(tag)
       if options.tag_relative && !stdout? && path.index('/') != 0
         Pathname.new(path).relative_path_from(tag_file_dir).to_s
       else
-        path
+        Pathname.new(path).cleanpath.to_s
```

Comment:
Well, to me, "simplest" is definitely "call a built-in that does that for you", rather than "write a regex with more slashes than actually significant characters: `/^\.\//`" :). 

But yeah, I can't think of any issues with removing the initial `./`, that would probably do the trick. Would you prefer I did that?